spec/defines/005_elasticsearch_instance_spec.rb

```diff
@@ -832,30 +837,35 @@ class { 'elasticsearch':
             should contain_file('/etc/elasticsearch/es-instance/jvm.options')
               .with_content(%r{
                 -Dfile.encoding=UTF-8.
+                -Dio.netty.allocator.numDirectArenas=0.
                 -Dio.netty.noKeySetOptimization=true.
                 -Dio.netty.noUnsafe=true.
                 -Dio.netty.recycler.maxCapacityPerThread=0.
                 -Djava.awt.headless=true.
+                -Djava.io.tmpdir=\${ES_TMPDIR}.
                 -Djna.nosys=true.
                 -Dlog4j.shutdownHookEnabled=false.
                 -Dlog4j2.disable.jmx=true.
                 -XX:\+AlwaysPreTouch.
                 -XX:\+HeapDumpOnOutOfMemoryError.
-                -XX:\+PrintGCDateStamps.
-                -XX:\+PrintGCDetails.
-                -XX:\+PrintTenuringDistribution.
                 -XX:\+UseCMSInitiatingOccupancyOnly.
                 -XX:\+UseConcMarkSweepGC.
```

Comment:
This should have an `8:' too

Reply:
I realize that the *test* isn't the only change but the point was the `-XX:+UseConcMarkSweepGC` should have an `8:` prefix as it was deprecated in 9.

Reply:
This has been added.